### DISCUSSION TITLE
Cant access web interface

### DISCUSSION THREAD

**Discussion by dorkalifa** [Security-Onion-Solutions/securityonion#13122] [Unsupported Versions]
Hello everyone,
The environment is the onionsec is a VM, Im accessing in through vpn, with the required firewall policies, 
i cant access the web interface after installing it, added any ip (0.0.0.0/0) to the firewall, the browser error is ERR_CONNECTION_RESET,
Though - if i telnet the address with 443 port, its listening, couldnt find any help in google, any suggestions?

**Comment by dougburks** [comment]
To clarify, have you allowed your IP address through Security Onion's host-based firewall?
https://docs.securityonion.net/en/2.4/firewall.html#configuring-host-firewall

**Comment by dorkalifa** [comment]
Hey doug, thanks for the answer,
I allowed any ip, added 0.0.0.0/0=any,
is adding any ip(0.0.0.0/0) or disabling the firewall is possible?
due, if i was blocked by the internal firewall, how can i telnet <ipaddr>:443?
